includes/class-wc-gateway-amazon-payments-advanced.php

```diff
@@ -709,13 +709,19 @@ public function display_payment_method_selected( $checkout_session = null ) {
 			if ( empty( $payments ) ) {
 				$change_label = esc_html__( 'Select', 'woocommerce-gateway-amazon-payments-advanced' );
 			}
+			$selected_label = esc_html__( 'Your selected Amazon payment method', 'woocommerce-gateway-amazon-payments-advanced' );
+			foreach ( $checkout_session->paymentPreferences as $pref ) { // phpcs:ignore WordPress.NamingConventions
+				if ( isset( $pref->paymentDescriptor ) ) { // phpcs:ignore WordPress.NamingConventions
+					$selected_label = $pref->paymentDescriptor; // phpcs:ignore WordPress.NamingConventions
```

Comment:
shouldn't concatenate here since is inside a foreach?

Reply:
I don't know in which case it can contain more than 1 item. That's why i did it that way.

Reply:
but it lost the original text ("Your selected Amazon payment method") too, is a total replacement?

Reply:
Yes. It was requested on slack.